Assignment2/bingyu-g-fromRDFtoJSON-LD.jsonld

```diff
@@ -0,0 +1,43 @@
+{
+	"@context":{
+		"as":"http://www.example.org/",
+		"rdfs": "http://www.w3.org/2000/01/rdf-schema#",	
+		"Class01":"as:Class01",
+		"Sensor029":"Class01:Sensor029",
+		"Measurement8401":"Sensor029:Measurement8401",
+		"Computer101":"Class01:Computer101",
+		"User10A":"Computer101:User10A",
+		"includes": "as:includes",
+		"hasMeasurement":"as:hasMeasurement",
+		"hasTemperature":"as:hasTemperature",
+		"hasName":"as:hasName",
+		"hasOwner":"as:hasOwner"
+	},
+	
+	"Class01": {"@id":"http://www.example.org/class01",
```

Comment:
Please remove "Class01":, "Sensor029":, "Measurement8401":, "Computer101": and "User10A": because they are resources not relationships. You only need the content defined between the curly braces that allow you to define their data.

Reply:
Please, remove the names that I listed before but those that are outside the context, I mean those that are on the data side.